.github/workflows/reproducible.yml

```diff
@@ -0,0 +1,46 @@
+name: "Reproducible build"
+on:
+  release:
+    types:
+      - created
+
+jobs: 
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Chekcout
+        uses: actions/checkout@v2
+
+      - name: Set version
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+
+      - name: Install packages
+        run: sudo apt-get update -y && sudo apt-get install -y git curl gnupg
+
+      - name: Download Release key
+        run: gpg --keyserver https://secchannel.rsk.co/release.asc --recv-keys 1A92D8942171AFA951A857365DECF4415E3B8FA4
+
+      - name: Check fingerprint
+        run: gpg --finger 1A92D8942171AFA951A857365DECF4415E3B8FA4
+
+      - name: Verify SHA256SUM
+        run: gpg --verify SHA256SUMS.asc && sha256sum --check SHA256SUMS.asc
+
+      - name: Build powpeg-node
+        run: ./configure.sh && ./gradlew --no-daemon clean build -x test
+
+      - name: Calculate hashes
+        working-directory: build/libs
+        run: sha256sum * > hashes
+
+      - name: Print hashes
+        run: echo $(cat build/libs/hashes)
```

Comment:
`sha256sums * | tee hashes` will achieve the same effect in just one step. Also, `echo $(cat file)` is a bad idiom for `cat file`.

Reply:
fixed